app/controllers/agents_controller.rb

```
@@ -7,14 +7,71 @@ def index
+        affiliations: render_to_string(partial: "#{partial_path}/affiliations", locals: { agent: mock_agent }),
+        usages: render_to_string(partial: "#{partial_path}/usages", locals: { agent: mock_agent }),
+        actions: render_to_string(partial: "#{partial_path}/actions", locals: { agent: mock_agent })
+      }
+    end
+
+    render json: {
+      collection: agent_data,
+      recordsTotal: agents.totalCount,
+      recordsFiltered: agents.totalCount,
+    }
+  end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for ajax_agents_list is too high. [<17, 37, 12> 42.45/23]

(Metrics/AbcSize)